docs/example-scenario/apps/hpc-saas.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Computer-aided engineering through high performance computing on Azure
+description: <Article Description>
+author: alexbuckgit
+ms.date: 08/22/2018
+---
+
+# Computer-aided engineering through high performance computing on Azure
+
+This example scenario demonstrates delivery of a software-as-a-service (SaaS) platform built on the high-performance computing (HPC) capabilities of Azure. This scenario is based on an engineering software solution. However, the architecture is relevant to other industries requiring HPC resources such as image rendering, complex modeling, and financial risk calculation.
```

Comment:
Is this SaaS for internal resources or building a platform to be consumed by external customers?